rosbridge_server/launch/rosbridge_websocket_launch.xml

```diff
@@ -22,10 +24,14 @@
   <arg name="params_glob" default="" />
   <arg name="bson_only_mode" default="false" />
 
+  <arg name="respawn" default="false"/>
+
+  <arg unless="$(var bson_only_mode)" name="binary_encoder" default="default"/>
+
   <arg unless="$(var bson_only_mode)" name="binary_encoder" default="default"/>
```

Comment:
This line is doubled now. @sea-bass Is this even needed? This argument does not seem to be used anywhere

Reply:
I don't understand how i haven't see this.
The argument seems to be used in the rosbridge_websocket node

Reply:
The `bson_only_mode` is used but the `binary_encoder` defined here doesn't seem to be used